distribution/tests/endpoints.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+rm -rf alfresco-identity-service-$IDENTITY_VERSION
+unzip alfresco-identity-service-$IDENTITY_VERSION.zip
+
+EXISTENT_KEYCLOAK_INSTANCES=$(ps aux | grep "standalone" | awk '{print $2}' | head -n 2 )
+echo $EXISTENT_KEYCLOAK_INSTANCES
+kill $EXISTENT_KEYCLOAK_INSTANCES
+
+cd alfresco-identity-service-$IDENTITY_VERSION/bin
+
+echo "Starting the identity service"
+/bin/bash -c './standalone.sh &'
+
+sleep 15
```

Comment:
Hardcoded sleep is just asking for trouble. If this is to allow Identity Service to start up, should we not rather periodically poll `curl -v http://localhost:8080/` until it returns (or reaches a timeout)?